syntax_and_semantics/macros.md

```diff
@@ -290,6 +290,24 @@ Foo.new.describe #=> "Class is Foo"
 Foo.describe #=> "Class is Foo"
 ```
 
+## Method information
+
+When a macro is invoked you can access the method, the macro is in with a special instance variable: `@def`. The type of this variable is `Def` unless the macro is outside of a method, in this case it's `NilLiteral`.
```

Comment:
Could you add links to these types in the API docs?

Reply:
Did that and also linked to all the other types.